.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build the application
+        run: |
+          echo "Placeholder for actual build commands"
+          # Replace the above line with actual build commands (like npm run build, etc.)
```

Comment:
I know we plan to put stuff in here, but I would be more comfortable if this PR didn't include this build section at all ... I reckon you should keep this on your local system as a stash and you can re-add it when you get around to it, if necessary.

Reply:
One of the questions I ask myself when looking at placeholder code in a PR, is if I don't come back to this code for _whatever_ reason, would we want it there? i.e. does it give useful instruction to others? In this case, I would say not.

Reply:
I see what you mean and I agree, a placeholder code is a good idea, but maybe not if it's being productionised